patches/3.6.0.0/vnest_patch

```diff
@@ -0,0 +1,7 @@
+echo "ECS Vnest patching" >> /tmp/vnest-patch.out
```

Comment:
Everything in this file should be moved into the appropriate Ansible playbook or it will cause collisions.